source/core/sound.asm

```diff
@@ -40,23 +41,24 @@ SECTION "SOUND INTERFACE", ROM0
 ; Destroys: all
 PlaySound::
     ; Find available yeller
-    ld hl, wYellerStates + YELLER_SIZE * (MAX_NUM_YELLERS - 2)
```

Comment:
Just so you know, this change does affect behavior. A backwards search usually means last played sound takes priority, whereas a forwards search usually means first played sound takes priority. I had intended the former.